oddt/shape.py

```diff
@@ -0,0 +1,244 @@
+import numpy as np
+from numpy.linalg import norm
+import scipy.stats as stats
+
+
+def common_usr(molecule, ctd=None, cst=None, fct=None, ftf=None, atoms_type=None):
+    """Function used in USR and USRCAT function
+
+        Parameters
+        ----------
+        molecule : oddt.toolkit.Molecule
+            Molecule to compute USR shape descriptor
+
+        ctd : numpy array or None (default = None)
+            Coordinates of the molecular centroid
+            If 'None', the point is calculated
+
+        cst : numpy array or None (default = None)
+            Coordinates of the closest atom to the molecular centroid
+            If 'None', the point is calculated
+
+        fct : numpy array or None (default = None)
+            Coordinates of the farthest atom to the molecular centroid
+            If 'None', the point is calculated
+
+        ftf : numpy array or None (default = None)
+            Coordinates of the farthest atom
+            to the farthest atom to the molecular centroid
+            If 'None', the point is calculated
+
+        atoms_type : str or None (default None)
+            Type of atoms
```

Comment:
`atom_dict` columns, like 'ishydrophobe', 'isacceptor', are used to select atoms.